### PR TITLE
Updating Readme with installation instructions for plugins

### PR DESCRIPTION
**Pull request by saratvemulapalli**
Signed-off-by: Sarat Vemulapalli <vemulapallisarat@gmail.com>

### Description
Updating Readme with installation instructions for plugins with OpenSearch
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,14 @@ We plan to remove the need for this work-around via [OpenSearch#581](https://git
 
 To upgrade your existing plugins to work with OpenSearch, see [UPGRADING](./UPGRADING.md).
 
+### Installing Plugins in OpenSearch
```
Comment (dblock) [inline README.md:47]: Update TOC?

Comment (saratvemulapalli) [reply README.md:47]: :) Nice catch. Sure.

File: README.md [hunk 2]
```diff
@@ -44,6 +45,14 @@ We plan to remove the need for this work-around via [OpenSearch#581](https://git
...
```
Comment (dblock) [inline README.md:48]: I think this is actually a bit unclear.

1) Are we telling people to assemble, extract and run OpenSearch? If so, this should say "Assemble, extract and run OpenSearch" and provide an example.

```
~/OpenSearch (main)> ./gradlew build
~/OpenSearch (main)> ./gradlew :distribution:archives:linux-tar:assemble
~/OpenSearch (main)> tar vfxz distribution/archives/linux-tar/build/distributions/opensearch-1.0.0-SNAPSHOT-linux-x64.tar.gz
~/OpenSearch (main)> cd opensearch-1.0.0-SNAPSHOT
~/OpenSearch/opensearch-1.0.0-SNAPSHOT (main)> ./bin/opensearch
```

Then, to build and install a plugin, for example job-scheduler:

```
~/job-scheduler (main)>  ./gradlew  assemble -Dbuild.version_qualifier=beta1 -Dbuild.snapshot=false
```

```
~/OpenSearch/opensearch-1.0.0-SNAPSHOT (main)> ./bin/opensearch-plugin install file://<your path>/build/distributions/opensearch-job-scheduler-1.0.0.0-beta1.zip
-> Installing file://.../build/distributions/opensearch-job-scheduler-1.0.0.0-beta1.zip
-> Downloading file://.../build/distributions/opensearch-job-scheduler-1.0.0.0-beta1.zip
[=================================================] 100%   
-> Installed opensearch-job-scheduler
```

2) If we're telling people to download a distribution, we should say from where and also provide a working example.

Probably both?

Comment (dblock) [reply README.md:48]: Futhermore, many plugins like anomaly-detection are not useful without dashboards if I am not mistaken. Let's explain how to get that onto here as well?

Comment (saratvemulapalli) [reply README.md:48]: Thats a great point. I started off to answer how can we install plugins but to work with a specific version I didnt think about that. But I definitely agree that would help. 

I'll take some of these changes and add them to the PR.

Comment (saratvemulapalli) [reply README.md:48]: Not really, plugins do have APIs but they are way more easier to have Dashboards plugins to use a UI. 
But I do agree, will have instructions to have both the plugins installed.

File: README.md [hunk 3]
```diff
@@ -44,6 +45,65 @@ We plan to remove the need for this work-around via [OpenSearch#581](https://git
...
+~/OpenSearch (main)> git checkout 1.0.0-beta1
+~/OpenSearch (main)> ./gradlew :distribution:archives:linux-tar:assemble -Dbuild.version_qualifier=beta1 -Dbuild.snapshot=false
+~/OpenSearch (main)> tar vfxz distribution/archives/linux-tar/build/distributions/opensearch-1.0.0-beta1-linux-x64.tar.gz
+~/OpenSearch (main)> cd opensearch-1.0.0-beta1
+~/OpenSearch/opensearch-1.0.0-beta1 (main)> ./bin/opensearch
+```
+
+Checkout, build and install the plugin.
+
+_Example_: Install Anomaly Detection(Job Scheduler plugin which is a dependency).
```
Comment (dblock) [inline README.md:67]: Install Anomaly Detection and Job Scheduler plugins

**Review by dblock — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (saratvemulapalli) [reply README.md:67]: :/ I missed this comment. Will send out another change later.

Comment (dblock) [reply README.md:67]: nbd

**Pull request merged**